docs/install.mdx

```diff
@@ -5,295 +5,80 @@ title: Installation
-
-</TabItem>
-</Tabs>
-
-## 3. Install KubeVela Core
-
-<Tabs
-    className="unique-tabs"
-    defaultValue="vela"
-    values={[
-        {label: 'Default', value: 'vela'},
-        {label: 'Helm', value: 'helm'},
-    ]}>
-<TabItem value="vela">
-
-> The version of vela CLI >= 1.2.3.
+expected output:
 
-```shell script
-vela install
 ```
+Preparing K3s images...
+Successfully prepare k3s image:  /Users/sunjianbo/.vela/velad/k3s/k3s-airgap-images-amd64.tgz
+Successfully prepare k3d images
 
-<details> <summary> check out the outcome </summary>
-
-```shell
-...
-Watching for changes to Job kubevela-vela-core-cluster-gateway-tls-secret-patch with timeout of 18m0s
-Add/Modify event for kubevela-vela-core-cluster-gateway-tls-secret-patch: ADDED
-kubevela-vela-core-cluster-gateway-tls-secret-patch: Jobs active: 0, jobs failed: 0, jobs succeeded: 0
-Add/Modify event for kubevela-vela-core-cluster-gateway-tls-secret-patch: MODIFIED
-kubevela-vela-core-cluster-gateway-tls-secret-patch: Jobs active: 1, jobs failed: 0, jobs succeeded: 0
-Add/Modify event for kubevela-vela-core-cluster-gateway-tls-secret-patch: MODIFIED
-kubevela-vela-core-cluster-gateway-tls-secret-patch: Jobs active: 1, jobs failed: 0, jobs succeeded: 0
-Add/Modify event for kubevela-vela-core-cluster-gateway-tls-secret-patch: MODIFIED
-Starting delete for "kubevela-vela-core-admission" ServiceAccount
-Starting delete for "kubevela-vela-core-admission" ClusterRole
-Starting delete for "kubevela-vela-core-admission" ClusterRoleBinding
-Starting delete for "kubevela-vela-core-admission" Role
-Starting delete for "kubevela-vela-core-admission" RoleBinding
-Starting delete for "kubevela-vela-core-admission-patch" Job
-Starting delete for "kubevela-vela-core-cluster-gateway-admission" ServiceAccount
-Starting delete for "kubevela-vela-core-cluster-gateway-admission" Role
-Starting delete for "kubevela-vela-core-cluster-gateway-admission" RoleBinding
-Starting delete for "kubevela-vela-core-cluster-gateway-tls-secret-patch" Job
+...snip...
 
 KubeVela control plane has been successfully set up on your cluster.
-If you want to enable dashboard, please run "vela addon enable velaux"
-```
-
-</details>
-
-</TabItem>
+If you want to enable dashboard, please run "vela addon enable /Users/sunjianbo/.vela/addons/velaux"
 
-<TabItem value="helm">
+Keep the token below if you want to restart the control plane
+K1075e7f6c77555e6ebdaf4854e4a2f39ae4287cfad23f27cdac5b33608d44633fe::server:zSpzbdbGzVxOwfBvvjgT
 
-If you are helm user, you can also use helm to install kubevela core:
+🚀  Successfully install KubeVela control plane
+🔭  See available commands with `vela help`
+```
 
-> helm v3.2.0+ required
+**Export the Kubernetes config **
 
 ```
-helm repo add kubevela https://charts.kubevela.net/core
-helm repo update
-helm install --create-namespace -n vela-system kubevela kubevela/vela-core --version 1.3.4 --wait
+export KUBECONFIG=~/.kube/velad-cluster-default
```

Comment:
Given `vela` installed by `velad install` will automatically read default velad cluster kubeconfig, This section can be:

**Optional** if you setup KubeVela by running `velad install` without parameters.

```
export KUBECONFIG=$(velad kubeconfig --host)
```